### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by styoti**
YM-23045

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+  - React-jsiexecutor (0.63.4):
+    - DoubleConversion
+    - Folly (= 2020.01.13.00)
+    - glog
+    - React-cxxreact (= 0.63.4)
+    - React-jsi (= 0.63.4)
+  - React-jsinspector (0.63.4)
+  - react-native-yoti-face-capture (0.1.0):
+    - React-Core
+    - YotiFaceCapture (= 3.0.0-alpha6)
```
Comment (AdrienVioletYoti) [inline example/ios/Podfile.lock:192]: You need to migrate to `3.0.0`

File: README.md [hunk 2]
```diff
@@ -1 +1,285 @@
...
+# Yoti Face Capture for React Native
+An easy to use face detection component for React Native from Yoti.
+Face detection is performed with the front facing camera, whose frames the library analyzes and then produces an optimised cropped image of the captured face.
+
+[![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/getyoti/react-native-yoti-face-capture?label=latest%20release)](https://github.com/getyoti/react-native-yoti-face-capture/releases) [![Publish Release](https://github.com/getyoti/react-native-yoti-face-capture/workflows/Publish%20Release/badge.svg)](https://github.com/getyoti/react-native-yoti-face-capture/actions?query=workflow%3A%22Publish+Release%22)
+<br>
+[![Platform - Android](https://img.shields.io/badge/platform-Android-3ddc84.svg?style=flat&logo=android)](https://www.android.com)
+[![Platform - iOS](https://img.shields.io/badge/platform-iOS-000.svg?style=flat&logo=apple)](https://developer.apple.com/ios)
+
+The library leverages [Google ML Kit](https://firebase.google.com/docs/ml-kit/detect-faces) for the face detection.
```
Comment (AdrienVioletYoti) [inline README.md:10]: That is only true for Android, for iOS we use the Vision library https://developer.apple.com/documentation/vision

File: README.md [hunk 3]
```diff
@@ -1 +1,285 @@
...
+- `FaceCaptureAnalysisErrorFaceNotCentered`
+- `FaceCaptureAnalysisErrorFaceNotStraight` (depending on configuration) 
+- `FaceCaptureAnalysisErrorFaceAnalysisFailed`
+- `FaceCaptureAnalysisErrorMultipleFaces`
+
+# Troubleshooting
+
+<details>
+	<summary>Resolving autolinking failures on Android and iOS</summary>
+
```
[the remark this answers is not in the document]
Comment (styoti) [reply README.md:225]: Thanks Marios!
The tags allow us to create a collapsible section, as these instructions will not apply to the majority of our integrators, and as such are better if they are not visible by default.
Alternatively, we can expand them in full and do away with the tags as they are already at the bottom of the page?
I'm happy with any approach here.
cc @AdrienVioletYoti

[the remark this answers is not in the document]
Comment (AdrienVioletYoti) [reply README.md:225]: Fine to keep the collapsible section.

File: example/ios/File.swift [hunk 4]
```diff
@@ -0,0 +1,6 @@
+//
+//  File.swift
```
[the remark this answers is not in the document]
Comment (styoti) [reply example/ios/File.swift:2]: Ok great.
It comes standard with the boilerplate for the example app so I'd kept it. I tested and indeed its absence has no effect 👍. I'll remove it.

**Review by AdrienVioletYoti — APPROVED** [review]

**Pull request merged**